build/webpack.base.conf.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 	resolve: {
 		extensions: ['.mjs', '.js', '.vue', '.json'],
 		alias: {
+			'~': resolve('node_modules'),
```

Comment:
@eddieferrer In your testing did this addition affect any of the image imports? Homepage components would be a quick place to check.

Reply:
No it does not affect them, everything still works.